tools/harness-thci/OpenThread.py

```diff
@@ -1362,6 +1362,7 @@ def reset(self):
             raise AssertionError("Could not connect with OT device {} after reset.".format(self))
 
         self.log('factoryreset finished within 10s timeout.')
+        self.__executeCommand('log level 5')
```

Comment:
This will raise a command error exception for platforms not enabling `OPENTHREAD_CONFIG_LOG_LEVEL_DYNAMIC_ENABLE`.

Reply:
That's true. Maybe use it for BR only.

Reply:
That would be okay.